### DISCUSSION TITLE
[Docs] release 日志请增加 breaking change

### DISCUSSION THREAD

**Discussion by ishowman** [element-plus/element-plus#17458] [Feature Request]
Feature type: **`Docs`**

## Details

- Enhanced component name: `N/A`

## Description

https://github.com/element-plus/element-plus/pull/13263  该改动属于 breaking change，但release 日志中是放在 refactor，容易被忽略

### Additional comments

_(empty)_

**Comment by shizhihuaxu** [comment]
有很多组件样式的破坏性变更也没有提及，版本变更不遵循语义化版本控制规范实在难顶
